webapp/pages/post/_id/_slug/index.vue

```diff
@@ -206,49 +215,70 @@ export default {
   },
 }
 </script>
-
 <style lang="scss">
-.page-name-post-id-slug {
+.post-page {
+  &.--blur-image > .ds-card-image img {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-03T17:26:52Z" title="Tuesday, December 3rd 2019, 6:26:52 pm +01:00">Dec 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -125,6 +151,7 @@ export default {
       ready: false,
       title: 'loading',
       showNewCommentForm: true,
+      blur: false,
```


```suggestion
      blurred: false,
```